docs/src/index.md

```diff
@@ -6,6 +6,40 @@ A Julia package containing a number of algorithms for analyzing images and autom
 Depth = 2
 ```
 
+## Getting started
+This package is part of a wider Julia-based image processing
+[ecosystem](https://github.com/JuliaImages). If you are a starting out, then you
+may benefit from [reading](https://juliaimages.org/latest/quickstart/) about
+some fundamental conventions that the ecosystem utilizes that are markedly
+different from how images are typically represented in OpenCV, MATLAB, ImageJ or
+Python.
+
+The usage examples in the `ImageBinarization.jl` package assume that you have
+already installed some key packages. Notably, it assumes that you are able to
+load and display an image. Loading an image is facilitated through the
+[FileIO.jl](https://github.com/JuliaIO/FileIO.jl) package, which uses
+[QuartzImageIO.jl](https://github.com/JuliaIO/QuartzImageIO.jl) if you are on `MacOS`, and
+`ImageMagick.jl` otherwise. Depending on your particular system configuration,
+you might encounter problems installing the image loading packages, in which
+case you can refer to the [troubleshooting guide](https://juliaimages.org/latest/troubleshooting/#Installation-troubleshooting-1).
+
+Image display is typically handled by the [ImageView.jl](https://github.com/JuliaImages/ImageView.jl) package. However, there are some known issues with this package. For example, on
+`Windows` the package has the side-effect of introducing substantial [input lag](https://github.com/JuliaImages/ImageView.jl/issues/176) when typing in the Julia REPL. Also, as of writing, some users
+of `MacOS` are [unable to use](https://github.com/JuliaImages/ImageView.jl/issues/175) the ImageView package.
+
+As an alternative, one can display an image using the [Makie.jl](https://github.com/JuliaPlots/Makie.jl) plotting package. There is also the [ImageShow.jl](https://github.com/JuliaImages/ImageShow.jl) package which facilitates displaying images in `Jupyter` notebooks
+via [IJulia.jl](https://github.com/JuliaLang/IJulia.jl).
```

Comment:
The two paragraphs above don't seem to be wrapped correctly. There are some line breaks, but not at the same length (~80 characters) used in the rest of the added text.

Reply:
Thank you